hack/generate-allure-report.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -xe
+
+ALLURE_VERSION=2.13.5
+ALLURE_DIR=out/allure
+ALLURE_BIN=$ALLURE_DIR/allure-$ALLURE_VERSION/bin
+
+if [ -z $JAVA_HOME ]; then
+    mkdir -p out
+    curl -s -L https://download.java.net/java/GA/jdk15.0.1/51f4f36ad4ef43e39d0dfdbaf6549e32/9/GPL/openjdk-15.0.1_linux-x64_bin.tar.gz -o out/java.tar.gz
+    tar -C out -xvf out/java.tar.gz
+    export JAVA_HOME=$(readlink -m out/jdk-15.0.1)
+fi
```

Comment:
I am not sure if people would like to install java just for being able to visualize reports.. IMHO, it is a too hard expectation. Instead, how about crafting a Dockerfile that would have all tools needed in it, and then use it from this script to generate reports?

Check https://github.com/fescobar/allure-docker-service

Reply:
Good idea, thx... done